release-notes.txt

```diff
@@ -1,3 +1,628 @@
++ xtimer: introduce xtimer_msleep()
++ usbus: Add USB peripheral serial string support
++ hashes: Add PBKDF2-sha256 implementation
++ usbus/dfu: add Device Firmware Upgrade support for USBUS
++ USB HID minimal implementation
++ Add helpers for pointer tagging
++ atomic_utils: Functions for atomic access
++ entropy_source: add new module for entropy sources
++ nimble_autoadv: add new module
+* ztimer:
+    + add ztimer_mutex_lock_timeout()
+    * fix xtimer_compat
+    * increase xtimer_compat coverage
+* oneway-malloc: check calloc args
+* event: Allow single event_thread handling all queues
+- irq_handler: Drop module
+
+Networking
+----------
++ gnrc_netif: add capability to join or leave link layer multicast groups
++ gnrc_sixlowpan_frag:
+    + initial import of Minimal Forwarding according to RFC 8930
+    + initial import of Selective Fragment Recovery (RFC 8931)
++ gnrc_netif_ethernet: Support RX timestamp
++ Initial implementation of IEEE 802.15.4 security
+* gnrc_pkt: refactor to not use utlist.h
+* sys/net/gnrc: fix in-band signaling of no RSSI
+* gnrc_netif_pktq: fix pktbuf leak
+* Fixed a critical bug, allowing to crash the whole system with a specially
+  crafted LoRa frame
+* GNRC LoRaWAN:
+    + Support for IN865 region, Improve debugging
+    * GNRC LoRaWAN: Bug fix for zero CFList
+* IEEE 802154 Radio HAL:
+    + add `read` function
+    + add IEEE802154_RADIO_INDICATION_CRC_ERROR
+    + add IEEE802154_RADIO_INDICATION_TX_START
+    + add helpers for RSSI and dBm conversion
+    * fix wrong rssi conversions treewide
+* IEEE 802.15.4 SubMAC:
+    + add retransmission reporting
+    * enable ACK_REQ by default
+    * unify legacy and radio HAL bootstrap
+* gcoap:
+    + Process CON responses
+    * make blocksize configurable
+- gcoap: remove deprecated gcoap_add_qstring
+- gnrc_nettype: remove GNRC_NETTYPE_IOVEC
+- gnrc_pktbuf: drop gnrc_pktbuf_replace_snip()
+- gnrc_sixlowpan_rb: remove GNRC_SIXLOWPAN_FRAG_RBUF_AGGRESSIVE_OVERRIDE
+- sock/dtls: remove sock_dtls_session_create due to deprecation
+
+Packages
+--------
++ openthread: add support for radios compatible with SubMAC
++ uwb_core: initial support
+* Updates:
+    * littlefs2: Bump to v2.3
+    * jsmn: bump version
+    * libb2: update to latest release
+    * libhydrogen: bump version
+    * micro-ecc: bump version
+    * tinycrypt: bump version
+    * fatfs: bump version to r0.14
+    * gecko_sdk: update to v2.7.7
+    * hacl: switch upstream to backup copy
+    * lvgl: bump to v7.7.1
+* openwsn:
+    + add ieee802154_hal based radio
+    + openwsn: add sock_udp
+- nordic-softdevice: remove deprecated package
+
+Boards
+------
+* Renode:
+    + stm32f4discovery: Add initial renode support
+    + firefly: add renode configuration
+    + hifive*: add renode support
+* Arduino feature:
+    + waspmote-pro: add arduino feature
+    + remote-revb: add arduino feature
++ New boards:
+    + ESP32-Ethernet-Kit
+    + arduino-nano-33-iot
+    + weact-f4x1cx
```

Comment:
```suggestion
    + weact-f401cc / weact-f401ce / weact-f411ce
```
might be more recognizable

Reply:
done! 2902039f8777cb08745e45e6f33afc3440bc6e08